src/styles/theme/oc-radio.scss

```diff
@@ -12,13 +7,17 @@
 @mixin hook-form-radio {
   @include oc-form-check-size(1);
 
-  background-color: $input-background;
-  border: 1px solid $input-border;
+  background-color: var(--oc-input-background);
+  border: 1px solid var(--oc-input-border);
 }
 
 .oc-radio {
   @extend .uk-radio;
 
```

Comment:
Could you change the default style so that the radiobutton uses the same input background like the checkbox? Should be solved by moving the background-color and border from the `hook-form-radio` mixin above to the `.oc-radio` class:

```
background-color: var(--oc-input-background);
border: 1px solid var(--oc-input-border);
```

Reply:
Also, could you add the same focus style that the checkbox already has? It's not necessarily in scope for this PR, but a quick win.